generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs

```diff
@@ -36,9 +36,11 @@ if (field.fieldValidate === true) {
     }
     if (rules.includes('min')) {
         validators.push('min="' + field.fieldValidateRulesMin + '"');
```

Comment:
Remove this, it doesn't do anything

Reply:
no it's use by some browser. Chrome add a slicer limited by min max, try it here : https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_max_min